dist/README.md

```diff
@@ -66,7 +70,7 @@ If everything looks fine, you may interrupt Compose (hit Ctrl-C), run
 `docker compose down` and use systemd to handle the Grist Docker process:
 
 ```sh
-sudo systemd enable --now grist
+sudo systemctl enable --now grist
```

Comment:
Minor, a little further down there are commands given which if followed literally run into a confusing auth prompt. Running with `sudo` works and I guess people would think of that?

```
grist@ip-172-31-81-246:~$ systemctl stop grist
==== AUTHENTICATING FOR org.freedesktop.systemd1.manage-units ====
Authentication is required to stop 'grist.service'.
Multiple identities can be used for authentication:
 1.  Ubuntu (ubuntu)
 2.  ,,, (grist)
Choose identity to authenticate as (1-2): 1
Password: 
polkit-agent-helper-1: pam_authenticate failed: Authentication failure
```

Reply:
Thanks, I added a couple more sudos.